client/state/login/reducer.js

```diff
@@ -51,7 +74,17 @@ const updateTwoStepNonce = ( state, { twoStepNonce, nonceType } ) => Object.assi
 
 export const twoFactorAuth = createReducer( null, {
 	[ LOGIN_REQUEST ]: () => null,
-	[ LOGIN_REQUEST_SUCCESS ]: ( state, { data, rememberMe } ) => data ? { ...data, remember_me: rememberMe } : null,
+	[ LOGIN_REQUEST_SUCCESS ]: ( state, { data } ) => {
+		if ( data ) {
+			const rest = omit( data, 'redirect_to' );
```

Comment:
Because this is the only data returned by the login endpoint that doesn't relate to 2FA:

```json
{
  "success": true,
  "data": {
    "redirect_to": "https:\/\/haiku.blog\/",
    "user_id": 12345,
    "two_step_supported_auth_types": ["backup", "sms", "push"],
    "two_step_nonce_backup": "obc8zkxbt1n9hk6zwbaijoa8ioi54y9h",
    "two_step_nonce_sms": "qs7qdntx5afn4dp65khc2juytzx7y32z",
    "two_step_nonce_push": "gv8n6nvws1ojrh6h4c1bacrwlk6g3pql",
    "push_web_token": "1234567890:ju5ps6nl1h0ys5umz0tszphyk1kjs7x4tlsyr4f5nbo6sm2lpdzowi9l3x2j0kf4",
    "two_step_notification_sent": "push"
  }
}
```